bogenliga/bogenliga-business-logic/src/main/java/de/bogenliga/application/business/Schusszettel/impl/business/SchusszettelComponentImpl.java

```diff
@@ -468,4 +468,12 @@ private String getMannschaftsNameByID(long mannschaftID){
         }
         return mannschaftName;
     }
+
+    // returns a dynamic font size, dependant on the length of the given text and the intended font size
+    private float getDynamicFontSize(String text, Float fontSize){
+        if(text.length() < 15){
+            return  fontSize;
+        }
+        return 175f  / text.length();
```

Comment:
Magic number: woher kommt die 175f?

Reply:
hier habe ich für die Berechnung eine hyperbole Funktion verwendet, die mir die Schriftgröße passen zurückgegeben hat
ich kann die Funktion ebenfalls in eine Variable auslagern und sinnvoll bennen?

Reply:
Auslagern muss nicht sein (erhöht nur die Komplexität) - lieber einen kurzen Kommentar darüber und gut ists